### PR TITLE
CLDR-17612 cla: update text, record version

### PR DESCRIPTION
**Pull request by srl295**
- update text of CLA
- improve format of CLA form
- also, fix date serialization issue in UserSettings (Gson setting issue) for the signed date

CLDR-17612

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

## Screenshot
image

### CODE REVIEW

**Comment by srl295** [comment]
fyi @annebright

**Comment by srl295** [comment]
text update:

image

**Review by AEApple — APPROVED** [review]
Preview LGTM, but someone else will need to review the java.

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserSettings.java [hunk 1]
```diff
@@ -92,7 +93,7 @@ public boolean persistent() {
     }
 
     public void setJson(String name, Object o) {
-        final Gson gson = new Gson();
+        final Gson gson = new GsonBuilder().setDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").create();
```
Comment (srl295) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserSettings.java:96]: this makes it work properly serializing dates.

**Review by btangmu — APPROVED** [review]

**Pull request merged**